.github/workflows/4-deploy-to-prod.yml

```diff
@@ -0,0 +1,103 @@
+name: Deploy to prod
+on:
+  push:
+    branches:
+      - 'releases/prod'
+
+jobs:
+
+  tag:
+    runs-on: ubuntu-latest
+    outputs:
+      new_tag: ${{ steps.github-tag-action.outputs.new_tag }}
+      changelog: ${{ steps.github-tag-action.outputs.changelog }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Bump and tag
+        id: github-tag-action
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: .*
```

Comment:
Should this be made `releases/prod`?

Reply:
I don't _think_ so but I'm not certain. The documentation can be found here:

https://github.com/mathieudutour/github-tag-action#filter-branches

My read on this is that it's a separate action-level filter. So the workflow definition could trigger on _any_ branch and if you wanted to limit the branches that this tag job applied to then you could specify branches here in this argument.

Thus in this action's current state, the job only kicks off from `releases/prod` and then when the tagging step happens `.*` matches `releases/prod` so the tagging action proceeds.